include/fluent-bit/flb_aws_util.h

```diff
@@ -147,8 +147,11 @@ flb_sds_t flb_aws_error(char *response, size_t response_len);
  * Similar to 'flb_aws_error', except it prints the JSON error type and message
  * to the user in a error log.
  * 'api' is the name of the API that was called; this is used in the error log.
+ *
+ * Returns 0 if an error cannot be found in the response,
+ * and 1 if the error is found and printed.
  */
-void flb_aws_print_error(char *response, size_t response_len,
+int flb_aws_print_error(char *response, size_t response_len,
```

Comment:
Why these return values?? Let's use either typical 0 success and -1 failure or FLB_TRUE and FLB_ERROR